src/AppInstallerCLICore/ExecutionReporter.cpp

```diff
@@ -120,6 +122,22 @@ namespace AppInstaller::CLI::Execution
             m_progressBar->ShowProgress(current, maximum, type);
         }
     }
+    
+    void Reporter::BeginProgress()
+    {
+        GetBasicOutputStream() << VirtualTerminal::Cursor::Visibility::DisableShow;
+        ShowIndefiniteProgress(true);
+    };
+
+    void Reporter::EndProgress(bool hideProgressWhenDone)
+    {
+        ShowIndefiniteProgress(false);
+        GetBasicOutputStream() << VirtualTerminal::Cursor::Visibility::EnableShow;

```

Comment:
>GetBasicOutputStream() << VirtualTerminal::Cursor::Visibility::EnableShow; [](start = 8, length = 74)

EnableShow might need to happen after the call to end progress on the progress bar.  It might not matter, but it seems like it will be better to maintain the ordering from the header.